Dockerfile-base

```diff
@@ -0,0 +1,23 @@
+FROM python:3.12-slim
+WORKDIR /usr/src/app
+
+# set environment variables
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+
+# Install pip and dependencies
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    git \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install dependencies
+RUN pip install git+https://github.com/AI4quantum/maestro.git@v0.2.0
+
+RUN chown -R 1000:100 /usr/src/app &&\
+    mkdir -p /usr/src/app/src/media && chown 1000:100 /usr/src/app/src/media
+
+EXPOSE 8000
+USER 1000:100
+
+# ENTRYPOINT ["maestro"]
```

Comment:
is there a reason to comment this out? could we uncomment this and not need the separate `Dockerfile-cli` below?